File: 프로그래머스/0/120882. 등수 매기기/등수 매기기.js
function solution(score) {
    let answer = new Array(score.length).fill(1)
    const avg = score.map(e => (e[0] + e[1]) / 2); 
    for (let i = 0; i < avg.length; i++) {
        for (let j = 0; j < avg.length; j++) {
            if (avg[i] < avg[j]) {
                answer[i]++;
            }
        }
    }
    return answer;
}

File: 프로그래머스/0/120882. 등수 매기기/등수 매기기.py
def solution(score):
    answer = [1] * len(score)
    avg = [(s[0] + s[1]) / 2 for s in score]
    
    for i in range(len(avg)):
        for j in range(len(avg)):
            if avg[i] < avg[j]:
                answer[i] += 1
    return answer